### PR TITLE
GEOMESA-2809 Upgrade dependencies to mitigate CVE findings

### PR DESCRIPTION
**Pull request by elahrvivaz**
* jackson 2.9.7 -> 2.9.10
* netty 4.1.17 -> 4.1.48
* commons-compress 1.18 -> 1.20

Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

**Comment by jnh5y** [comment]
We need CQs, right?  (Once there's a GeoTools release, we'll need a round of CQs for that as well.  Maybe we can batch those together.)

File: build/cqs.tsv [hunk 1]
```diff
@@ -9,10 +9,10 @@ com.conversantmedia:disruptor	1.2.13	compile
 com.drewnoakes:metadata-extractor	2.8.1	compile
 com.esotericsoftware:kryo-shaded	3.0.3	compile
 com.esotericsoftware:minlog	1.3.0	compile
-com.fasterxml.jackson.core:jackson-annotations	2.9.7	compile
-com.fasterxml.jackson.core:jackson-core	2.9.7	compile
-com.fasterxml.jackson.core:jackson-databind	2.9.7	compile
-com.fasterxml.jackson.dataformat:jackson-dataformat-cbor	2.9.7	compile
+com.fasterxml.jackson.core:jackson-annotations	2.9.10	compile
+com.fasterxml.jackson.core:jackson-core	2.9.10	compile
```
Comment (elahrvivaz) [inline build/cqs.tsv]: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21839

File: build/cqs.tsv [hunk 2]
```diff
@@ -9,10 +9,10 @@ com.conversantmedia:disruptor	1.2.13	compile
 com.drewnoakes:metadata-extractor	2.8.1	compile
 com.esotericsoftware:kryo-shaded	3.0.3	compile
 com.esotericsoftware:minlog	1.3.0	compile
-com.fasterxml.jackson.core:jackson-annotations	2.9.7	compile
-com.fasterxml.jackson.core:jackson-core	2.9.7	compile
-com.fasterxml.jackson.core:jackson-databind	2.9.7	compile
-com.fasterxml.jackson.dataformat:jackson-dataformat-cbor	2.9.7	compile
+com.fasterxml.jackson.core:jackson-annotations	2.9.10	compile
```
Comment (elahrvivaz) [inline build/cqs.tsv]: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21838

File: build/cqs.tsv [hunk 3]
```diff
@@ -9,10 +9,10 @@ com.conversantmedia:disruptor	1.2.13	compile
...
+com.fasterxml.jackson.core:jackson-core	2.9.10	compile
+com.fasterxml.jackson.core:jackson-databind	2.9.10.3	compile
```
Comment (elahrvivaz) [inline build/cqs.tsv]: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21840

File: build/cqs.tsv [hunk 4]
```diff
@@ -9,10 +9,10 @@ com.conversantmedia:disruptor	1.2.13	compile
...
+com.fasterxml.jackson.dataformat:jackson-dataformat-cbor	2.9.10	compile
```
Comment (elahrvivaz) [inline build/cqs.tsv]: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21841

File: build/cqs.tsv [hunk 5]
```diff
@@ -70,9 +70,9 @@ io.dropwizard.metrics:metrics-ganglia	3.2.6	compile
 io.dropwizard.metrics:metrics-graphite	3.2.6	compile
 io.dropwizard.metrics:metrics-jvm	3.2.6	compile
 io.github.azagniotov:dropwizard-metrics-cloudwatch	1.0.13	compile
-io.netty:netty-all	4.1.17.Final	compile
-io.netty:netty-buffer	4.1.17.Final	compile
-io.netty:netty-common	4.1.17.Final	compile
+io.netty:netty-all	4.1.48.Final	compile
```
Comment (elahrvivaz) [inline build/cqs.tsv]: covered by https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21836

File: build/cqs.tsv [hunk 6]
```diff
@@ -70,9 +70,9 @@ io.dropwizard.metrics:metrics-ganglia	3.2.6	compile
...
+io.netty:netty-buffer	4.1.48.Final	compile
+io.netty:netty-common	4.1.48.Final	compile
```
Comment (elahrvivaz) [inline build/cqs.tsv]: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21843

File: build/cqs.tsv [hunk 7]
```diff
@@ -70,9 +70,9 @@ io.dropwizard.metrics:metrics-ganglia	3.2.6	compile
 io.dropwizard.metrics:metrics-graphite	3.2.6	compile
 io.dropwizard.metrics:metrics-jvm	3.2.6	compile
 io.github.azagniotov:dropwizard-metrics-cloudwatch	1.0.13	compile
-io.netty:netty-all	4.1.17.Final	compile
-io.netty:netty-buffer	4.1.17.Final	compile
-io.netty:netty-common	4.1.17.Final	compile
+io.netty:netty-all	4.1.48.Final	compile
+io.netty:netty-buffer	4.1.48.Final	compile
```
Comment (elahrvivaz) [inline build/cqs.tsv]: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21842

File: build/cqs.tsv [hunk 8]
```diff
@@ -126,7 +126,7 @@ org.apache.avro:avro	1.7.5	compile
 org.apache.camel:camel-core	2.24.1	compile
 org.apache.camel:camel-scala	2.24.1	compile
 org.apache.commons:commons-collections4	4.4	compile
-org.apache.commons:commons-compress	1.18	compile
+org.apache.commons:commons-compress	1.20	compile
```
Comment (elahrvivaz) [inline build/cqs.tsv]: existing cq here, but not yet approved: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21771

File: build/cqs.tsv (hunk 1 above)
Comment (elahrvivaz) [reply build/cqs.tsv]: resolved

File: build/cqs.tsv (hunk 2 above)
Comment (elahrvivaz) [reply build/cqs.tsv]: approved

File: build/cqs.tsv (hunk 7 above)
Comment (elahrvivaz) [reply build/cqs.tsv]: approved

File: build/cqs.tsv (hunk 3 above)
Comment (elahrvivaz) [reply build/cqs.tsv]: approved

File: build/cqs.tsv (hunk 8 above)
Comment (jnh5y) [reply build/cqs.tsv]: Approved!

File: build/cqs.tsv (hunk 4 above)
Comment (elahrvivaz) [reply build/cqs.tsv]: approved

File: build/cqs.tsv (hunk 6 above)
Comment (elahrvivaz) [reply build/cqs.tsv]: approved

**Comment by jnh5y** [comment]
merged as 1095d445a792ae041ad206c660614e8a1f1a2b05.

**Pull request closed without merging**